pkg/ccl/kvccl/kvtenantccl/tenant_upgrade_test.go

```diff
@@ -170,14 +169,13 @@ func TestTenantUpgrade(t *testing.T) {
 			"SHOW CLUSTER SETTING version",
 			[][]string{{clusterversion.TestingBinaryVersion.String()}})
 	})
-
 }
 
-// Returns three versions :
-//   - v0 corresponds to the bootstrapped version of the tenant,
-//   - v1, v2 correspond to adjacent releases.
+// Returns two versions v0, v1, v2 which correspond to adjacent releases. v0 will
```

Comment:
Is this change future-proof?  If, for example, we let MinSupportedVerion trail a few releases, would this comment still be correct?  I'd _imagine_ that the test would still do what we'd want it to do, though maybe not if we wanted to test all of the upgrade stages in between v0 and v2.